contracts/package.json

```diff
@@ -67,11 +67,13 @@
     "release:major": "scripts/publish.sh major"
   },
   "devDependencies": {
-    "@nomicfoundation/hardhat-chai-matchers": "^1.0.6",
+    "@nomicfoundation/hardhat-chai-matchers": "^2.0.8",
+    "@nomicfoundation/hardhat-ethers": "^3.0.8",
     "@nomicfoundation/hardhat-network-helpers": "^1.0.8",
-    "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers@0.3.0-beta.13",
-    "@typechain/ethers-v5": "^10.2.0",
-    "@typechain/hardhat": "^6.1.5",
+    "@nomiclabs/hardhat-solhint": "^4.0.1",
+    "@openzeppelin/contracts": "^5.1.0",
+    "@typechain/ethers-v6": "^0.5.1",
```

Comment:
The current code doesn't have incomplete migration, nothing breaks except the watcher.ts which can be ignored for now as it's not used. Although some imports can be used from `ethers` instead of `ethersproject` but it only makes sense if `ethersproject` can be replaced with `ether` and that's out of scope for this PR.